components/dependencies.mdx

```diff
@@ -1,59 +1,105 @@
 ---
-title: "Dependencies"
-description:
-  "Architect was designed with the future in mind, and the future of any team
-  building distributed software is a complex web of services."
+title: Dependencies
+description: Integrate with other APIs using common dependency management tactics
 ---
 
-Historically this complex web of services has been hard to manage, hard to
-reason about, and hard for developers to contribute to.
+Historically cloud teams have tried to avoid dependencies and coupling between microservices and APIs
+because it creates a lot of infrastructure and CI/CD complexity. However, its all too natural for
+developers to try to extend what exists rather than reinvent it, and many teams ended up with complex
+webs of microservices nonetheless. 
 
-Fortunately, the graphing and collaboration problems this creates have been
-solved before. We've seen it with object-oriented programming and inheritance,
-and even more recently with package and dependency management. All of our
-favorite languages have a way for developers to utilize dependencies through a
-resolver to handle the artifact storage, complex graphing, and de-duplication
-needed to manage them at-scale. Architect takes this same approach to make it
-easier than ever for developers to extend cloud services.
+Fortunately, this problem of "dependencies" has been solved before. Every language and operating system
+on earth has tools for dependency management to help developers collaborate and extend each others work.
+Now Architect is here to provide the same value for APIs and microservices.
 
-## Utilizing dependencies
+## Why use dependencies?
 
-```yaml
+Citing dependencies as part of your component is not the most natural way for developers to integrate with
+software made by other teams, but its also the only way to achieve [on-demand environments](/environments/pull-request)
+for [distributed applications](/architecture/microservices).
+
+Every time a component is deployed with Architect, it will automatically deploy (if needed) and connect to
+cited dependencies. Perhaps even more impressive than that is that it will also deploy the dependencies OF your
+component's dependencies and ensure they're integrated as well.
+
+Without dependency management integrated into deployments, each and every application would have to maintain a
+complex web of CI pipelines or centralized infrastructure-as-code (IaC) templates to create new environments which
+can be an educational and logistical nightmare. With Architect, all you ever need to know is your immediate dependencies
+and the rest will be automated for you.
+
+## Registering dependencies
+
+Before we can begin using dependencies, we need to create a component that we want to act as a dependency for another.
+Let's start by creating a new component that we'll call the `backend` of our project. The command below will help you
+create a new component from one of Architect's starter projects.
+
+```sh
+$ architect init backend
```

Comment:
This command needs more instruction. We need to explain what they should select from the dropdown list and that a new project will be created in the directory provided (backend). We should explain that this command is cloning one of our starter projects locally.

Reply:
I'm not sure which to select. We don't publish the `node-rest-api` project as a starter project any longer. We publish the `nodejs` project, which contains a frontend, instead.

Reply:
I see what you mean, but I think if we verbosely explain the `init` command every time we cite it we're going to have a LOT of duplicate documentation. 

What I think makes more sense given your two comments is not to make this a full "guide" with running examples, but rather to simply talk about the dependencies feature abstractly and the `register` and `link` command respectively. If you agree I can make some changes to shorten.